org-roam-buffer.el

```diff
@@ -108,8 +108,9 @@ When non-nil, the window will not be closed when deleting other windows."
             (key-backlinks (org-roam--get-backlinks (s-chop-prefix "cite:" roam-key)))
             (grouped-backlinks (--group-by (nth 0 it) key-backlinks)))
       (progn
-        (insert (format "\n\n* %d Cite backlinks\n"
-                        (length key-backlinks)))
+        (insert (let ((l (length key-backlinks)))
+                  (format "\n\n* %d Cite backlink%s\n"
+                          l (if (> l 1) "s" ""))))
```

Comment:
```suggestion
                          l (when (> l 1) "s"))))
```

Is this idiomatic Lisp? (and does it work?)

Reply:
`format` would replace `%s` with `nil` (i.e. literal `nil`) if we used `when`.  Empty-string is not the same as `nil` in that regard.

Reply:
True, `nil` is formatted as `"nil"`.